resolver-functions/SampleForReport.ts

```diff
@@ -79,14 +80,6 @@ export const SampleForReportResolver = async (root, args, context) => {
                           path
                         }
                       }
-                      accession {
-                        accessionId
-                        accessionName
-                      }
-                      taxon {
-                        id
-                        name
-                      }
                       sequencingRead {
```

Comment:
@rainandbare are we querying for the sequencing read technology here rather than at the top level because it's just easier to process the data that way?